templates/base.html

```diff
@@ -30,6 +30,24 @@
   </head>
 
   <body>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light container">
+        <div class="container-fluid col">
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+              <a class="nav-link active" aria-current="page" href="#"> Leaderboard</a>
+              <a class="nav-link" href="#">My Points</a>
+              <a class="nav-link" href="#" tabindex="-1" aria-disabled="true">Help</a>
+            </div>
+          </div>
+        </div>
+        <a class="navbar-brand col text-end" href="#">Game Name</a>
```

Comment:
I think you could add the `href="/"` so that clicking the name brings you to the homepage again.

Reply:
Ok